docker/Dockerfile

```diff
@@ -115,6 +115,7 @@ ARG DISABLE_LIBARCHIVE
 ARG DISABLE_LZMA
 ARG DISABLE_BZIP2
 ARG DISABLE_SCRIPTING
+ARG DISABLE_PYTHON_SCRIPTING
```

Comment:
DockerBuild.sh and DockerConfig.cmake should be updated as well to pass this parameter

Reply:
Updated.